core/src/button/button.tsx

```diff
@@ -188,24 +199,42 @@ interface ButtonComponent
 		outset: ButtonStyle;
 		flat: ButtonStyle;
 	};
+	color: {
+		highlight: ButtonColor;
+		danger: ButtonColor;
+	};
 }
 
+const getColor = (props: ButtonProps) => {
+	if (props.highlight !== undefined && props.color !== undefined)
+		throw Error(
+			"highlight and color cannot be defined at the same time. Use color only"
```

Comment:
Should be put attribute name in the backtick? I think it will be easier to scan.

![image](https://user-images.githubusercontent.com/30283022/125103744-dedcd580-e106-11eb-9838-246baeb4cd82.png)